class.json-api-endpoints.php

```diff
@@ -1139,7 +1141,8 @@ function get_author( $author, $show_email = false ) {
 			$avatar_URL = $this->api->get_avatar_url( $email );
 		}
 
-		$email = $show_email ? (string) $email : false;
+		$email = $show_email_and_ip ? (string) $email : false;
+		$ip_address = $show_email_and_ip ? (string) $ip_address : false;
```

Comment:
These could be grouped to check the condition once:
```php
if ( $show_email_and_ip ) {
    $email = (string) $email;
    $ip_address = (string) $ip_address;
} else {
    $email = '';
    $ip_address = '';
}
```
Also, why setting them to `false`, a `boolean` type, when they're not meant to be shown? Why not consistently use a `string` type, an empty string?